.github/workflows/unit-tests.yml

```diff
@@ -30,3 +30,22 @@ jobs:
 
     - name: Run tests
       run: ./test.sh
+      
+  prod:
+      
+      runs-on: ubuntu-latest
+      
+      needs: test
+      
+      steps:
+      - uses: actions/checkout@v2
+      
+      - name: Login to Docker Hub
+        run: echo ${{ secrets.DOCKER_HUB_PASSWORD }} | docker login -u ${{ secrets.DOCKER_HUB_USERNAME }} --password-stdin
+        continue-on-error: true
+
+      - uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true
+        
+      - name: Build production image
+        run: docker build --target listenbrainz-prod --build-arg deploy_env=test --build-arg GIT_COMMIT_SHA=test .
```

Comment:
by setting the build args to these values, it means that we can't reuse the docker cache from the test build on the base image. Consider setting them to the same value as we do in docker-compose.yml

Reply:
I have removed the deploy env variable and fixed some shellcheck warnings as well.